setup/bindir/cloud-setup-databases.in

```diff
@@ -443,7 +444,7 @@ for example:
 
             self.encryptiontype = self.options.encryptiontype
             self.mgmtsecretkey = self.options.mgmtsecretkey
-            self.dbsecretkey = self.options.dbsecretkey
+            self.dbsecretkey = shlex.quote(self.options.dbsecretkey)
```

Comment:
@Spaceman1984 this adds a quote to the string, why do we want to add that? Are we passing it as a parameter to something on shell?

Reply:
Yes, the ampersand breaks a call to the shell. Refer back to the error on #4925

Reply:
Pl see the usage in `encrypt()` and add quotes there.